### DISCUSSION TITLE
Add extensions field to response

### DISCUSSION THREAD

**Discussion by m0rtalis** [MichalLytek/type-graphql#1460] [Q&A] [answered]
Is it possible to add an `extensions` field to the response as described in the GraphQL Spec https://spec.graphql.org/October2021/#sec-Response-Format?

**Comment by MichalLytek** [comment, marked as answer]
It's out of scope of TypeGraphQL. It's a code-first schema builder, not a GraphQL server.

**Reply by m0rtalis** [reply]
Thanks. Then I will have to look how I can achieve it with Apollo Server
